lib/common.js

```diff
@@ -75,16 +74,7 @@ module.exports = {
         asyncArrow: "always",
       },
     ],
-    "comma-dangle": [
-      "error",
-      {
-        arrays: "always-multiline",
-        objects: "always-multiline",
-        imports: "never",
-        exports: "never",
-        functions: "never",
-      },
-    ],
+    "comma-dangle": ["error", "always-multiline"],
```

Comment:
Why this change? Curious on the `functions` one?

Reply:
If we are going to trailing comma, might as well trailing comma everything.

Okay I'll come clean, it's how `prettier` behaves by default 😅 

I'll let you decide whether you want to incorporate it

Reply:
I just don't want functions to do:

```


/*eslint comma-dangle: ["error", {"functions": "always"}]*/

function foo(a, b,) {
}

foo(a, b,);
new foo(a, b,);
```